### PR TITLE
Phase 1 : Home page cleaning (PR changes).

### PR DESCRIPTION
**Pull request by walahamad**
This PR contains the cleaning home page PR (https://github.com/ibatta/CBISmoke/pull/203) changes and fix an issue in menu validation.

### CODE REVIEW

File: src/com/generic/page/Cart.java [hunk 1]
```diff
@@ -127,7 +127,7 @@ public static String getTotalPrice() throws Exception {
 	}
 
 	//Done CBI
-	//TODO:: Duplicated from HomePage file.
+	//TODO:: Remove this function and use the isListLoaded from HomePage class (Duplicated from HomePage file).
 	public static boolean isListDisplayed(List<WebElement> elements) {
```
Comment (ibatta) [inline src/com/generic/page/Cart.java]: let us remove this function and do changes on cart and run the cart tests again

File: src/com/generic/page/Cart.java [hunk 2]
```diff
@@ -138,7 +138,7 @@ public static boolean isListDisplayed(List<WebElement> elements) {
 	}
 
 	//Done CBI
-	//TODO:: Duplicated from HomePage file.
+	//TODO:: Remove this function and use the isListLoaded from HomePage class (Duplicated from HomePage file).
```
Comment (ibatta) [inline src/com/generic/page/Cart.java]: let us remove this function and do changes on cart and run the cart tests again

File: src/com/generic/page/Cart.java (hunk 1 above)
Comment (walahamad) [reply src/com/generic/page/Cart.java]: Done in Cleaning CLP PR.

File: src/com/generic/page/Cart.java (hunk 2 above)
Comment (walahamad) [reply src/com/generic/page/Cart.java]: Done in Cleaning CLP PR.

**Pull request merged**